sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -88,15 +88,15 @@
     "@azure/core-rest-pipeline": "1.0.0-beta.2",
     "@azure/core-lro": "^1.0.2",
     "@azure/core-paging": "^1.1.1",
-    "@azure/core-tracing": "1.0.0-preview.10",
+    "@azure/core-tracing": "1.0.0-preview.11",
     "@azure/logger": "^1.0.0",
-    "@opentelemetry/api": "^0.10.2",
+    "@opentelemetry/api": "^0.18.0",
     "tslib": "^2.0.0"
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
-    "@azure/identity": "^1.1.0",
+    "@azure/identity": "2.0.0-beta.1",
```

Comment:
same question about the caret.

Reply:
I think you're right about this one, I'll add in the caret.

My reasoning is that @sadasant  would want to know, immediately, if his changes in his preview surface will not work with each library. We'll still need to do a general sweep when he GAs his preview but at least this would prevent a lot of flux in between.

Reply:
Caret added (no stick)